src/wallets/EthersWallet.ts

```diff
@@ -129,10 +140,12 @@ export class EthersWallet implements TLWallet {
     }
 
     if (!('TLWalletVersion' in deserializedWallet)) {
-      // Use the old serialising method if no TL_WALLET_VERSION key in the deserialized wallet
-      encryptedKeystore = serializedWallet
+      // Use the old serializing method if no TL_WALLET_VERSION key in the deserialized wallet
```

Comment:
If you want to be able to load the old format for backwards compatibility, it should be in loadAccount, otherwise I would remove it.

Reply:
Ok so it would be okay for you if I omit backwards compatibility for now? So I would keep version 1 of the TLWallet but include breaking changes as we have no real user currently anyways.